### ISSUE TITLE
BN span & compliance loging

### ISSUE THREAD

**Issue by nvpstr** [mlcommons/training_policies#133]
We should **report BN span** (the number of samples over which BN stats are computed for normalization during training). This is an important model hyper-parameter and it may differ from the batch-size.

**Comment by petermattson** [comment]
SWG:
Tentatively acceptable as non-required tag. Could also provide in readme. NVIDIA to add tag by Tuesday. Confined to SSD and ResNet.

**Comment by petermattson** [comment]
Make a required tag in 5.1.

**Comment by nvpaulius** [comment]
It is a required tag in v0.6
